app/admin-tab/host/controller.js

```diff
@@ -19,27 +13,29 @@ export default Ember.Controller.extend({
   custom: Ember.computed.equal('customRadio', 'yes'),
   editing: true,
   saving: false,
-  customValue: '',
 
-  looksPublic: function() {
-    return isPublic(this.get('activeValue'));
-  }.property('activeValue'),
+  customHostValue: '',
 
-  activeValue: function() {
-    if ( this.get('custom') )
-    {
-      return this.get('customValue').trim();
-    }
-    else
-    {
-      return this.get('thisPage');
-    }
-  }.property('custom','customValue','thisPage'),
 
   actions: {
+    setActiveValue: function(value) {
```

Comment:
This can just be replaced with the `mut` helper in the template for settings that don't need special logic (like the catalog one does).